txscript/error.go

```diff
@@ -269,6 +274,7 @@ var errorCodeStrings = map[ErrorCode]string{
 	ErrDiscourageUpgradableNOPs: "ErrDiscourageUpgradableNOPs",
 	ErrNegativeLockTime:         "ErrNegativeLockTime",
 	ErrUnsatisfiedLockTime:      "ErrUnsatisfiedLockTime",
+	ErrNullFail:                 "ErrNullFail",
```

Comment:
Please add this at the same relative position as it was added to the definitions for consistency.  I know the compiler doesn't care, but it's nice to be consistent for us mere humans.